server/src/inmantals/server.py

```diff
@@ -321,25 +345,41 @@ async def textDocument_didSave(self, **kwargs):  # noqa: N802
     async def textDocument_didClose(self, **kwargs):  # noqa: N802
         pass
 
-    def convert_location(self, loc):
+    def convert_location(self, target):
         prefix = "file:///" if os.name == "nt" else "file://"
-        if isinstance(loc, Range):
+        if isinstance(target, AnchorTarget) and isinstance(target.location, Range):
```

Comment:
Why is there the `isinstance(target, AnchorTarget)` 

I assume all locations are pre-converted? 
if not, the else branch will still be broken I think? (e.g. assuming `target` is a` Range`)

Reply:
Ow I forgot to clean this up, this is form the implementation before I noticed it was not backward compatible. all locations are pre-converted indeed